test/metabase/driver_test.clj

```diff
@@ -41,7 +44,7 @@
     (is (driver/available? "metabase.driver-test/test-driver")
         "`driver/available?` should work for if `driver` is a string -- see #10135")))
 
-(deftest unique-connection-property-test
+(deftest ^:parallel unique-connection-property-test
```

Comment:
This means `tx.env/set-test-drivers!` should also trigger warnings in `:parallel` tests, right?

Reply:
Yes, but I'll do that as a follow-on